apps/ui/src/views/JoinView.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="container text-center mt-10">
     <h1>We hold weekly meetings on Tuesdays at
```

Comment:
Consider using another element, e.g. `div`, with styled text size to avoid screen reader/accessibility issues:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Accounted for.